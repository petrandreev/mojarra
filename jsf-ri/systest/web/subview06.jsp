<!--
 Copyright 2003 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->
<%@ page contentType="text/html" language="java" %>
<%@ taglib prefix="c" uri="http://java.sun.com/jstl/core" %>
<%@ taglib prefix="f" uri="http://java.sun.com/jsf/core" %>
<%@ taglib prefix="h" uri="http://java.sun.com/jsf/html" %>

<f:view>
<html>
<head>
<title>subview06</title>
</head>
<body>
<h:outputText value="Begin test <c:import> with iterator tag in imported page"/>
<f:subview id="subviewOuter">
<br>
<c:import url="subviewIterator01.jsp"/>
</f:subview>
<h:outputText value="Text from subview06.jsp"/>
<h:outputText value="End test <c:import> with iterator tag in imported page"/>
</body>
</html>
</f:view>
