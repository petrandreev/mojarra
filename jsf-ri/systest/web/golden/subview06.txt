

<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->






<html>
<head>
<title>subview06</title>
</head>
<body>
<p>Begin test &lt;c:import&gt; with iterator tag in imported page</p>

<br />
<p>

<!--
 Copyright 2004 Sun Microsystems, Inc. All rights reserved.
 SUN PROPRIETARY/CONFIDENTIAL. Use is subject to license terms.
-->







  <br>
  
    Array[0]: 
    This component has no ID <br>
    <input type="text" name="subviewOuter:subviewInner:j_id_id18pc2" value="This component has no ID " /><br>
  
    Array[1]: 
    This component has no ID <br>
    <input type="text" name="subviewOuter:subviewInner:j_id_id18pc2j_id_1" value="This component has no ID " /><br>
  
    Array[2]: 
    This component has no ID <br>
    <input type="text" name="subviewOuter:subviewInner:j_id_id18pc2j_id_2" value="This component has no ID " /><br>
  
    Array[3]: 
    This component has no ID <br>
    <input type="text" name="subviewOuter:subviewInner:j_id_id18pc2j_id_3" value="This component has no ID " /><br>
  



</p>


<p>Text from subview06.jsp</p>

<p>End test &lt;c:import&gt; with iterator tag in imported page</p>
</body>
</html>

